apps/fabric-website/src/pages/Controls/DrawerPage/DrawerPage.tsx

```diff
@@ -8,7 +8,14 @@ const baseUrl = 'https://github.com/OfficeDev/office-ui-fabric-react/tree/master
 
 export const DrawerPage: React.StatelessComponent<IControlsPageProps> = props => {
   const { platform } = props;
-  return <ControlsAreaPage {...props} title="Drawer" {...DrawerPageProps[platform]} otherSections={_otherSections(platform) as IPageSectionProps[]} />;
+  return (
+    <ControlsAreaPage
+      {...props}
+      title="Drawer"
+      {...DrawerPageProps[platform]}
+      otherSections={_otherSections(platform) as IPageSectionProps[]}
+    />
+  );
```

Comment:
Prettier change.

Reply:
I wonder if a bunch of these got introduced by the pre-commit script while running `git merge`? You should be able to revert them, and then use the `-n` option when committing.